README.md

```diff
@@ -41,6 +41,7 @@ A collection of awesome useful and awesome links, resources and shiny things for
     * [Linda](https://www.lynda.com/)
     * [Sololearn](https://www.sololearn.com/)
     * [Khan Academy](https://www.khanacademy.org/computing)
+    * [Skillshare](https://www.skillshare.com/)
```

Comment:
We'd love to have free websites/apps :smile: . But it seems that Skillshare has a free month

Reply:
Yes, a free month and if you refer a friend who signs up for Skillshare. Then you get another free month :smiley: